test/port-finder-integration-test.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * port-finder-0-vs-127-test.js: Test for the `portfinder` module.
+ * that demonstrates issue #24 
+ * https://github.com/indexzero/node-portfinder/issues/24
+ */
+
+var vows = require('vows'),
+    assert = require('assert'),
+    portfinder = require('../lib/portfinder'),
+    child_process = require('child_process'),
+    path = require('path'),
+    http = require('http');
+
+const host = "localhost";
```

Comment:
@winfinit - this pr looks good to go. Would you mind just changing this to var instead of const for jshint and doing a rebase. Then I pull the trigger. Thanks.